### DISCUSSION TITLE
can I still obtain Arctic Code Vault Contributor badges?

### DISCUSSION THREAD

**Discussion by dhans11** [community/community#138453] [Discussions] [answered]
### Select Topic Area

Question

### Body

I was wondering if it's still possible to obtain this badge ?

**Comment by vickybelario** [comment, marked as answer]
The Arctic Code Vault Contributor badge was awarded to GitHub users who had repositories or commits archived in the GitHub Arctic Code Vault snapshot, which took place on February 2, 2020. This was a one-time event, so if you did not have code in a repository archived by that date, you would not be able to obtain the badge now.
